src/universal/redux/getAuth.js

```diff
@@ -0,0 +1,76 @@
+import {cashay} from 'cashay';
```

Comment:
this is a magical little helper that gives me the user from the cashay store by typing something like `user = getAuth()`. i think this helper pattern is gonna be repeated when we use a team, or teammember, etc. i really like it.